config/prod.json

```diff
@@ -10,6 +10,10 @@
   "drsHubUrlRoot": "https://drshub.dsde-prod.broadinstitute.org",
   "firecloudBucketRoot": "https://storage.googleapis.com/firecloud-alerts",
   "firecloudUrlRoot": "https://portal.firecloud.org",
+  "externalCreds": {
+    "urlRoot": "https://externalcreds.dsde-dev.broadinstitute.org",
```

Comment:
Are you thinking we would test against dev? We'll need a reminder to switch this to the prod instance